Search/SearchQuery.php

```diff
@@ -85,4 +99,36 @@ public function setIndexes($indexes)
     {
         $this->indexes = $indexes;
     }
+
+    /**
+     * @return string
+     */
+    public function getOrder()
+    {
+        return $this->order;
+    }
+
+    /**
+     * @param string $order
+     */
+    public function setOrder($order)
+    {
+        $this->order = $order;
+    }
+
+    /**
+     * @return string
+     */
+    public function getSort()
+    {
+        return $this->sort;
+    }
+
+    /**
+     * @param string $sort
+     */
+    public function setSort($sort)
```

Comment:
This should be an array as follows:

````php
$query->setSort[
    'field1' => 'asc',
    'field2' => 'desc',
    'field3' => 'asc',
]);
```

In this way you can sort by multiple fields.